src/coreclr/src/jit/codegen.h

```diff
@@ -1028,6 +1028,7 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     void genHWIntrinsic_R_R_R_RM(
         instruction ins, emitAttr attr, regNumber targetReg, regNumber op1Reg, regNumber op2Reg, GenTree* op3);
     void genBaseIntrinsic(GenTreeHWIntrinsic* node);
+    void genX86BaseIntrinsic(GenTreeHWIntrinsic* node);
```

Comment:
We could probably just merge this with `genBaseIntrinsic` (I don't have a preference on which name is taken).

We've tried to keep these compressed as more things have become table driven.

Reply:
The main reason I added a new method was that everything in X86Base should be scalar, whereas most (all?) things in Vector are not.  I can recombine if you think we'd be able to make the codegen side table-driven, though.

Reply:
We can deal with it in a follow up PR. I'm not particularly concerned right now and it is a bigger refactoring to allow scalar intrinsics to be table driven in codegen.